plugins/catalog-react/src/hooks/useEntityPermission.ts

```diff
@@ -36,6 +43,14 @@ export function useEntityPermission(permission: Permission): {
   error?: Error;
 } {
   const { entity, loading: loadingEntity, error: entityError } = useEntity();
+
+  let defaultValue: boolean;
+  if (entity && cache[`${permission.name}${stringifyEntityRef(entity)}`]) {
```

Comment:
should really compute this key once :) and maybe put a space or some other character between them